src/main/features/core/lyrics/translate.ts

```diff
@@ -0,0 +1,50 @@
+import axios from 'axios';
+
+export const translateLyrics = async (
+    originalLyrics: string,
+    apiKey: string,
+    apiProvider: string | null,
+    targetLanguage: string | null,
+) => {
+    let TranslatedText = '';
+    if (apiProvider === 'Microsoft Azure') {
+        try {
+            const response = await axios({
+                data: [
+                    {
+                        Text: originalLyrics,
+                    },
+                ],
+                headers: {
+                    'Content-Type': 'application/json',
+                    'Ocp-Apim-Subscription-Key': apiKey,
```

Comment:
One issue users might run into is that if the translator resource [is created using a selected region instead of global](https://learn.microsoft.com/en-us/azure/ai-services/translator/reference/v3-0-reference#authenticating-with-a-regional-resource), it will require an additional header value for `Ocp-Apim-Subscription-Region` or the API will return HTTP 401.

This can be handled by either:
- Adding an additional setting when using Azure translator to select the region
- Or -- adding a note in the description that the resource needs to be created using a global region scope rather than regional

I'd be ok with either.

Reply:
> One issue users might run into is that if the translator resource [is created using a selected region instead of global](https://learn.microsoft.com/en-us/azure/ai-services/translator/reference/v3-0-reference#authenticating-with-a-regional-resource), it will require an additional header value for `Ocp-Apim-Subscription-Region` or the API will return HTTP 401.
> 
> This can be handled by either:
> 
> * Adding an additional setting when using Azure translator to select the region
> * Or -- adding a note in the description that the resource needs to be created using a global region scope rather than regional
> 
> I'd be ok with either.

Thanks for the review, I would probably choose to just make a clear note about selecting the global scope. Both Google and Microsoft recommend using the global scope as it automatically chooses the closed data center. Thus, I think it might be just more beneficially for us common users to use global scope as the connection could be more stable all around the globe? By the way, I also rechecked doc about the specific region scopes of both Google and Microsoft, there just like hundreds of scopes and various second-level domain based on different scope (e.g. Microsoft: america: api.nam.*, asia pacific: api-ap.*, europe: api-eur.*, and for switzerland, even change in top-level domain. Google: europe: eu.*)

I'll submit a new commit real quick.